theme/theme.go

```diff
@@ -15,6 +15,10 @@ var (
 	BrightnessIcon = theme.NewThemedResource(brightnessIcon, nil)
 	// BrokenImageIcon is the material design icon for a broken image
 	BrokenImageIcon = theme.NewThemedResource(brokenImageIcon, nil)
+	// MaximizeIcon is the material design icon for maximizing a window
+	MaximizeIcon    = theme.NewThemedResource(maximizeIcon, nil)
+	// MinimizeIcon is the material design icon for minimizing a window
+	MinimizeIcon = theme.NewThemedResource(minimizeIcon, nil)
```

Comment:
In the API this is referred to as "Iconic" but here "Minimize" - we should be consistent...